ls1intum/Artemis
# Add assessment highlighting to Apollon

### Checklist

### Motivation and Context


Currently Apollon does not support highlighting the feedback within an exams second correction round

### Description

We add the possibility to highlight feedback correction rounds into the modeling assessment editor and Apollon. 
Please note: online mergeable / testable after Artemis uses Apollon > 2.7.0 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> Exam with second correction round -> modeling exercise second correction
3. You should see the Difference View Button. Clicking on it shows a new badge for every feedback type (Apollon + unreferenced Feedback). The Badge states which correction round the feedback was taken in. 
4. Please Note: Apollon Feedback Badges are only refreshed with new information once the button is switched off and on again due to Apollon Artemis communication

### Test Coverage




### Screenshots


<img width="658" alt="Bildschirmfoto 2021-06-01 um 07 57 08" src="https://user-images.githubusercontent.com/42205689/120273675-1c9c4200-c2af-11eb-8565-03382d72c178.png">
<img width="1394" alt="Bildschirmfoto 2021-06-01 um 08 03 43" src="https://user-images.githubusercontent.com/42205689/120274306-f1662280-c2af-11eb-8192-8d4e1bf4cd14.png">
